src/app/shared/components/filters/filters.component.ts

```diff
@@ -50,11 +53,17 @@ export class FiltersComponent {
     return FilterType;
   }
 
-  public get isFiltered() {
+  public get isFilterChangedSinceLastApply() {
+    return !_.isEqual(this._appliedFilters, this._filters);
```

Comment:
Can we avoid call function from getter? It may impact on the performance of the component because angular would evaluate it on each tick to determine is value changed or not.

Reply:
Yes, refactored